docs/install.md

```diff
@@ -134,6 +135,23 @@ Termux is an Android terminal application and Linux environment, which can also
 
 To upgrade run: `yarn global upgrade code-server --latest`
 
+### Known Search Issue
+
+There is a known issue with search not working on Android because it's missing `bin/rg`. To fix:
+
+1. Install `ripgrep` with `pkg`
+   ```sh
+   pkg install ripgrep
+   ```
+2. Make a soft link using `ln -s`
```

Comment:
doesn't matter at all, but using autonumbering might be useful?

Reply:
Assuming you're referring to the 2 steps i.e. 
1.
1.

Instead of 
1. 
2.

? I think it's fine!

Reply:
yea, and I agree it's fine as it is :)